src/Migrations/Version20240206145629.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * See BG2-2550
+ */
+final class Version20240206145629 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $campaignIdForBG22550 = '5041';
+
+        $this->addSql(<<<SQL
+            UPDATE Donation
+            SET
+                tbgGiftAidRequestQueuedAt = NULL,
+                tbgGiftAidRequestFailedAt = NULL,
+                tbgGiftAidRequestConfirmedCompleteAt = NULL,
+                tbgGiftAidRequestCorrelationId = NULL,
+                tbgGiftAidResponseDetail = NULL,
+                salesforcePushStatus = 'pending-update'
```

Comment:
We haven't updated salesforcePushStatus in similar tasks before, but I think we should now that we're going to put `tbgGiftAidRequestConfirmedCompleteAt` in Salesforce.